holoviews/tests/plotting/bokeh/test_pathplot.py

```diff
@@ -402,6 +401,20 @@ def test_contours_linear_color_op(self):
         self.assertEqual(cmapper.low, 3)
         self.assertEqual(cmapper.high, 7)
 
+    def test_contours_empty_path(self):
+        import pandas as pd
+        contours = Contours([
+            pd.DataFrame([], columns=['x', 'y', 'color', 'line_width']),
+            pd.DataFrame({'x': np.random.rand(10), 'y': np.random.rand(10),
+                          'color': ['red']*10, 'line_width': [3]*10},
+                         columns=['x', 'y', 'color', 'line_width'])
+        ], vdims=['color', 'line_width']).opts(
+            color='color', line_width='line_width')
+        plot = bokeh_renderer.get_plot(contours)
+        cds = plot.handles['source']
+        self.assertEqual(cds.data['color'], np.array(['red']))
```

Comment:
@philippjfr Should it be replaced by that?

```suggestion
        glyph = plot.handles['glyph']
        self.assertEqual(glyph.line_color, 'red')
        self.assertEqual(glyph.line_width, 3)
```

Reply:
Oh weird, thought that's what I pushed.